5_databases/week1/New_Query_homework.sql

```diff
@@ -0,0 +1,66 @@
+-- Active: 1706443440629@@127.0.0.1@3306@hyf
+
+--1.Find out how many tasks are in the task table
+SELECT COUNT(title) AS count_tasks
+FROM task
+
+--2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) AS count_unvaliddate 
```

Comment:
Typo: invalid date😉 I like that you've been using AS. My another  suggestion to you is choosing either `camelCase` or `snake_case` for better readability. And don't mix them😊